src/Parser.php

```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```

Comment:
yup indeed; depends what you want the scope of the project to be; it's your call as to exactly how much scope creep you want; was just an idea :D